Resources/Prototypes/_Nuclear14/Entities/Objects/Consumable/Flora/wastelandflora.yml

```diff
@@ -312,24 +312,30 @@
 
 - type: entity
   parent: N14WastelandFlora
-  id: N14WastelandFloraWildJalapenoPepper
-  name: wild jalapeno pepper plant
-  description: A jalapeno pepper plant growing wild. It's healthy.
+  id: N14WastelandFloraWildGlowpepper #N14 change
+  name: wild glowpepper plant #N14 change
+  description: A glowpepper plant GLOWING wild. AH HA HA HA... it's healthy. #N14 change
```

Comment:
Don’t need change comments :)

Reply:
Do I need to remove it? I left these on everything I changed and only remembered today I only needed them on upstream changes